tiledb/bioimg/converters/base.py

```diff
@@ -392,7 +392,7 @@ def _create_image_pyramid(
     compressor: tiledb.Filter,
     pyramid_kwargs: Mapping[str, Any],
 ) -> None:
-    with tiledb.open(base_uri, "r") as a:
+    with tiledb.open(base_uri, attr="intensity") as a:
```

Comment:
Is the specification of the attribute name necessary when opening an array for either read or write since we only have one attribute? Because in `line 260` the array is opened without that.

Reply:
In `L:260` the array is opened in `mode=w` so `attr` is not necessary there. The reason why we introduce this approach in `mode=r` is cause even with one attribute arrays the resulted slices of arrays are of type `ordered_dict`, if `attr` is not set during opening. If set though we get `numpy.arrays` as a return value and thus the introduction of the `attr` arg during opening does not cause too many code changes.

Reply:
Not sure if it's necessary for writes as well but in either case it's very unlikely that is not needed just in line 260. 

A good reminder of the value of [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself): `attr="intensity"` is repeated 6 times, plus a most-likely missing 7th. This **must** be DRY-fied.

Reply:
It is not needed in `L:409` as well, it is typo. No `write_op` cares about the array being opened with the `attr` value, since the `attr` value is given during the creation of the array schema. I get that in read mode we could dry-fy it but this would require a wrapper of `tiledb.open` which idk if it is the recommended way to go.

Reply:
A global read_kwargs wouldn't suffice? It would be defined once so changes would be in a single point but it should be referenced in every location we read.

Reply:
At the very least the hardcoded `"intensity"` should be replaced with a global var (e.g. `ATTR_NAME`) but IMO a `tiledb.open` wrapper makes sense here so that any reader/writer of the code shouldn't have to know or guess whether `attr` is necessary or not based on `mode`.